trajectories/rkffile.py

```diff
@@ -429,7 +429,7 @@ def read_frame(self, i, molecule=None):
             self.firsttime = False
 
         self.position = i
-        return self.coords, cell
```

Comment:
This surely should be a copy instead of a direct reference to the coordinates... otherwise when reading multiple frames the coords are changed as a side-effect of this method...

Reply:
Intentional - reverted